rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -120,10 +120,11 @@ static NEVER_INLINE s32 savedata_op(ppu_thread& ppu, u32 operation, u32 version,
 				if (entry.name.substr(0, prefix.size()) == prefix)
 				{
 					// Count the amount of matches and the amount of listed directories
-					if (listGet->dirListNum++ < setBuf->dirListMax)
+					listGet->dirNum++; // total number of directories
+					if (listGet->dirListNum < setBuf->dirListMax)
 					{
-						listGet->dirNum++;
-
+						listGet->dirListNum++; // number of directories in list
+																	 
```

Comment:
what the heck is this :)

Reply:
A bunch of whitespace. Sorry I did not see it locally in my diff.

Reply:
no worries. I just think it's funny